templates/hooks/table/sdk_read_one_post_set_output.go.tpl

```diff
@@ -0,0 +1,6 @@
+	if isTableCreating(&resource{ko}) {
+		return &resource{ko}, requeueWaitWhileCreating
+	}
+	if isTableUpdating(&resource{ko}) {
+		return &resource{ko}, requeueWaitWhileUpdating
+	}
```

Comment:
Why do we need the above code hooks in the `sdkReadOne` post-set-output code paths?

Reply:
Sometimes tables needs few seconds to get created. re-queuing here is needed to sync the `.status.TableStatus` field.

Reply:
Here what i get when i create a table and wait for few seconds
```shell
k get tables
NAME   TABLENAME   TABLESTATUS
tb1    tb1         CREATING
```

```shell
aws dynamodb describe-table --table-name tb1 | jq
{
    ...
    "TableStatus": "ACTIVE",
    ...
}
```

Reply:
This is fixing a very similar issue to what @phanimullapudi was seeing in mq controller https://github.com/aws-controllers-k8s/community/issues/831

Reply:
for reviewers: @A-Hilaly and I walked through the code and Amine patiently waited for me to understand he was totally correct. :)